personal/privacyidea/default/mfatype_webauthn/phase01.tpl

```diff
@@ -21,22 +21,40 @@
 
 <h2>{t}Confirm security key setup{/t}</h2>
 
-<div class="section">
-    <p>{t escape=no 1=$tokenDescription}Please confirm the setup of the security key <b>%1</b>.{/t}</p>
-    <p>{t}Your browser or operating system will guide you through the setup of the security key.{/t}</p>
+<div class="section row">
+    <div class="col m12 l6">
+        <div class="section">
+            <p>{t escape=no 1=$tokenDescription}Please confirm the setup of the security key <b>%1</b>.{/t}</p>
+            <p>{t}Your browser or operating system will guide you through the setup of the security key.{/t}</p>
 
-    <ol>
-        <li>{t}Connect your security key to your computer, e.g. plug it into a USB port or place it near the computer&apos;s NFC reader.{/t}</li>
-        <li>{t}Initiate the setup by pressing the button below.{/t}</li>
-        <li>{t}Confirm the setup using a device-specific mechanism, e.g. by touching a sensor on the key.{/t}</li>
-        <li>{t}The browser will proceed to the next page if the setup has been sucessful.{/t}</li>
-    </ol>
-    <button class="btn primary"
-        name="mfa_webauthn_confirm"
-        type="submit">{t}Start setup{/t}</button>
-</div>
+            <ol>
+                <li>{t}Connect your security key to your computer, e.g. plug it into a USB port or place it near the computer&apos;s NFC reader.{/t}</li>
+                <li>{t}Initiate the setup by pressing the button below.{/t}</li>
+                <li>{t}Confirm the setup using a device-specific mechanism, e.g. by touching a sensor on the key.{/t}</li>
+                <li>{t}The browser will proceed to the next page if the setup has been sucessful.{/t}</li>
+            </ol>
+            <button class="btn primary"
+                name="mfa_webauthn_confirm"
+                type="submit">{t}Start setup{/t}</button>
+        </div>
 
-<div class="card-paneli red red-lighten4 text-red text-darken-4" id="mfaWebAuthnError"></div>
+        <div class="card-panel red lighten-4 red-text text-darken-4" id="mfaWebAuthnError"></div>
```

Comment:
Why do we need a message box here anyway?

Reply:
It's in the the mockup: errors from Webauthn setup are supposed to be displayed here, if an error occurs the button should restart the setup.

Reply:
Ah okay.